cpp/ql/lib/semmle/code/cpp/Type.qll

```diff
@@ -1621,6 +1621,11 @@ class RoutineType extends Type, @routinetype {
    */
   Type getReturnType() { routinetypes(underlyingElement(this), unresolveElement(result)) }
 
+  /**
+   * Holds if this function type has "C" language linkage.
```

Comment:
This isn't strictly necessary, but it might be nice to clarify this and provide a target-language example.
> This includes any function declared in a C source file, or explicitly marked as having "C" linkage:
> \`\`\`
> extern "C" void myFunction();
> \`\`\`

Reply:
Good idea, done.